dissect/volume/md/md.py

```diff
@@ -0,0 +1,160 @@
+from __future__ import annotations
+
+import datetime
+import io
+import operator
+import struct
+from typing import BinaryIO, Union
+from uuid import UUID
+
+from dissect.util import ts
+from dissect.util.stream import RangeStream
+
+from dissect.volume.md.c_md import c_md
+from dissect.volume.md.stream import (
+    RAID0Stream,
+    RAID10Stream,
+    RAID456Stream,
+    create_linear_stream,
+)
+
+
+class MD:
```

Comment:
Can you add some generic class doc strings? So it is a bit more clear which class to use for what.